core/src/main/java/org/jdbi/v3/core/Cleanables.java

```diff
@@ -1,107 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.jdbi.v3.core;
-
-import java.io.Closeable;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
-
-import org.jdbi.v3.core.exception.UnableToCloseResourceException;
-import org.jdbi.v3.core.statement.StatementBuilder;
-import org.jdbi.v3.core.transaction.TransactionState;
-
-/**
- * Resource management for JDBI. Cleanables can be registered on a SQL statement and they get cleaned up when the
- * statement finishes or (in the case of a ResultIterator), the object representing the results is closed.
- *
- * Resources managed by JDBI are {@link ResultSet}, {@link Statement}, {@link Handle} and {@link StatementBuilder} for historical reasons.
- */
-class Cleanables implements Closeable
```

Comment:
All the methods in this class were inlined. `StatementContext` now owns the `Set<Cleanable>`